packages/amplify-e2e-tests/src/__tests__/auth_4.test.ts

```diff
@@ -52,7 +50,9 @@ describe('amplify updating auth...', () => {
     };
     await initAndroidProjectWithProfile(projRoot, { ...defaultsSettings, disableAmplifyAppCreation: false });
     await addAuthWithSignInSignOutUrl(projRoot, settings);
+    await amplifyPushAuth(projRoot);
     await updateAuthSignInSignOutUrl(projRoot, settings);
+    await expect(amplifyPushAuth(projRoot, true)).resolves.not.toThrowError();
```

Comment:
What's the purpose of wrapping the push call in an `expect`?

If anything throws, doesn't the test fail anyways?